src/components/NewCardForm.js

```diff
@@ -4,3 +4,90 @@ import emoji from 'emoji-dictionary';
 import './NewCardForm.css';
 
 const EMOJI_LIST = ["", "heart_eyes", "beer", "clap", "sparkling_heart", "heart_eyes_cat", "dog"]
+
+class NewCardForm extends Component{
+
+  constructor() {
+    super();
+
+    this.state = {
+      text: '',
+      emoji: ''
+    }
+  }
+
+  onInputChange = (event) => {
+    const newState = {};
+
+    newState[event.target.name] = event.target.value;
+
+    this.setState(newState);
+  }
+
+  resetState = () => {
+    this.setState({
+      text: '',
+      emoji: ''
```

Comment:
Good use of a helper method here. Could you call this from your constructor instead of repeating this work?